src/main/scala/org/zalando/spearheads/innkeeper/oauth/OAuthService.scala

```diff
@@ -40,7 +43,13 @@ class OAuthService @Inject() (val config: Config,
     Try {
       val json = Await.result(futureJson, 1.second)
       json.parseJson.convertTo[AuthorizedUser]
-    }.toOption
+    } match {
+      case Success(value) => Some(value)
+      case Failure(ex) => {
+        logger.warn("The OAuth exception is", ex)
+        None
+      }
+    }
```

Comment:
can be rewritten as:
`.transform(identity, {case e => logger.warn("The OAuth exception is", ex); Failure(e)}).toOption`